source/_posts/2020-09-19-summing-up-the-month-of-what-the-heck.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: Summing-up the month of 'What the heck?!'
+description: All good things come to an end, including the month of 'What the heck?!'. So, how was this first edition?
+date: 2020-09-19 00:00:00
+date_formatted: "September 19, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+comments: true
+categories: Announcements
+og_image: /images/blog/2020-08-18-the-month-of-what-the-heck/social.png
+---
+
+<a href='https://community.home-assistant.io/c/what-the-heck/52'><img src='/images/blog/2020-08-18-the-month-of-what-the-heck/social.png' style='border: 0;box-shadow: none;'></a>
```

Comment:
`class='no-shadow'` should work instead of border: 0
```suggestion
<a href='https://community.home-assistant.io/c/what-the-heck/52'><img src='/images/blog/2020-08-18-the-month-of-what-the-heck/social.png' class='no-shadow'></a>
```